manifests/charts/ztunnel/templates/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+{{- define "mesh" }}
+  defaultConfig:
+    proxyMetadata: {}
+{{- end }}
+
+{{- $originalMesh := include "mesh" . | fromYaml }}
+{{- $mesh := mergeOverwrite $originalMesh .Values.meshConfig }}
+
```

Comment:
I don't think we need any of this, just put `.Values.meshConfig` below

Reply:
Fixed.